docs/components/navs.md

```diff
@@ -14,7 +14,11 @@ Navigation available in Bootstrap share general markup and styles, from the base
 
 ## Base nav
 
-The base `.nav` component is built with flexbox and provide a strong foundation for building all types of navigation components. It includes some style overrides (for working with lists), some link padding for larger hit areas, and basic disabled styling. No active states are included in the base nav.
+The base `.nav` component is built with flexbox and provide a strong foundation for building all types of navigation components. It includes some style overrides (for working with lists), some link padding for larger hit areas, and basic disabled styling.
+
+{% callout info %}
+The base `.nav` component does not include any `.active` state. The following examples include the class, mainly to demonstrate that this particular class does not trigger any special styling.
```

Comment:
i think as this is a bit "meta" (in that it clarifies why all the following examples use `.active` but don't look any different) it would warrant being in a callout. otherwise it can and will easily be missed (see https://github.com/twbs/bootstrap/issues/22470)